client/app/bundles/course/assessment/submission/components/GetHelpChatPage/ConversationArea.tsx

```diff
@@ -99,6 +119,19 @@ const ConversationArea: FC<ConversationAreaProps> = (props) => {
           </div>
         );
       })}
+      {isCurrentThreadExpired && (
+        <div
+          className="justify-self-center cursor-pointer rounded-lg bg-gray-200 pt-3 pl-3 pr-3 pb-2 m-2 w-fit text-wrap break-words"
+          onClick={() => dispatch(resetLiveFeedbackChat({ answerId }))}
+        >
+          <Typography
+            className="whitespace-pre-wrap text-center hover:underline"
+            variant="body2"
+          >
+            {t(translations.threadExpired)}
+          </Typography>
+        </div>
```

Comment:
nit: the `hover:underline` should go with the div where the `onClick` is handled instead of the text, so that when the cursor hovers and becomes a pointer, the underline effect also takes place.

Reply:
done

Reply:
done